packages/ui/src/components/Slider/components/DoubleSlider.tsx

```diff
@@ -187,7 +189,7 @@ export const DoubleSlider = ({
   // Get slider size (for options)
   useEffect(() => {
     const setWidthResize = () => {
-      setWidth(refSlider.current?.offsetWidth ?? SLIDER_WIDTH.max)
+      setWidth(refSlider.current?.offsetWidth ?? 0)
```

Comment:
I find 0 to be confusing, can we not put `Number(refSlider.current?.offsetWidth)` so we know that this value will always be taken, as long as the useEffect set a default value